packages/code-du-travail-frontend/src/layout/Header/index.js

```diff
@@ -117,29 +113,26 @@ const LogoLink = styled.a`
   align-items: center;
   text-decoration: none;
   @media (max-width: ${breakpoints.mobile}) {
-    /**
-     * hack(lionelb)
-     * the width will be 50% + half the logo
-     * so logo is center using space-between
-     */
-    flex: 0 0 calc(50% + 4.635rem);
```

Comment:
mitigated about the loosing author reference, ti sometimes eaisier to ask when we forgot why we do some stuff

Reply:
though it only work because of space-between :)